src/responder/kcm/kcmsrv_ccache.h

```diff
@@ -72,6 +72,13 @@ errno_t kcm_cc_new(TALLOC_CTX *mem_ctx,
                    krb5_principal princ,
                    struct kcm_ccache **_cc);
 
+/*
+ * Return shallow copy of ccache. Only ccache and credentials are duplicated,
+ * but their data are not.
```

Comment:
As far as I can tell, this is no longer correct - the behavior has also been modified to deep copy the creds.